### ISSUE TITLE
Markers won't display in leaflet 2.0.0

### ISSUE THREAD

**Issue by JianWang2016** [rstudio/leaflet#527]
Hi,
   It appears to me markers won't display when using leaflet 2.0.0 from github.
Using the following sample codes, markers show up just fine under leaflet 1.1.0 (CRAN) but not in leaflet 2.0.0 (github).  2.0.0 is needed in order to use crosstalk.  My sessionInfo() for both leaflet 1.1.0 and 2.0.0 are appended.  I wonder if anyone has encountered the same issue.

data(quakes)

```r
leaflet(data = quakes[1:20,]) %>% addTiles() %>%
  addMarkers(~long, ~lat, popup = ~as.character(mag), label = ~as.character(mag))

```
leaflet 1.1.0
> sessionInfo()

R version 3.4.4 (2018-03-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Red Hat Enterprise Linux

Matrix products: default
BLAS: /opt/R/R-3.4.4/lib64/R/lib/libRblas.so
LAPACK: /opt/R/R-3.4.4/lib64/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] leaflet_1.1.0

loaded via a namespace (and not attached):
 [1] htmlwidgets_1.0 shiny_1.0.5     compiler_3.4.4  magrittr_1.5    R6_2.2.2       
 [6] htmltools_0.3.6 tools_3.4.4     yaml_2.1.18     Rcpp_0.12.16    crosstalk_1.0.1
[11] jsonlite_1.5    digest_0.6.15   xtable_1.8-2    httpuv_1.3.6.2  mime_0.5  

leaflet 2.0.0
> sessionInfo()
R version 3.4.4 (2018-03-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Red Hat Enterprise Linux

Matrix products: default
BLAS: /opt/R/R-3.4.4/lib64/R/lib/libRblas.so
LAPACK: /opt/R/R-3.4.4/lib64/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] leaflet_2.0.0   devtools_1.13.5

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.16    withr_2.1.2     digest_0.6.15   mime_0.5        R6_2.2.2       
 [6] xtable_1.8-2    jsonlite_1.5    git2r_0.21.0    magrittr_1.5    httr_1.3.1     
[11] curl_3.2        tools_3.4.4     htmlwidgets_1.0 crosstalk_1.0.1 shiny_1.0.5    
[16] httpuv_1.3.6.2  yaml_2.1.18     compiler_3.4.4  memoise_1.1.0   htmltools_

**Comment by schloerke** [comment]
@JianWang2016 Are you viewing in the browser or within RStudio?

**Comment by tim-salabim** [comment]
On my linux machine I see

![screenshot at 2018-04-17 17 35 04](https://user-images.githubusercontent.com/2826432/38880360-cc8b0cfa-4265-11e8-9225-6f7cbe9e8161.png)

**Comment by schloerke** [comment]
@JianWang2016  Ah! The markers are showing up, but the tiles are not.  At least for the next couple of days, install the patch so at least `addTiles()` works.  Still working on the issue of https url protocols in linux within RStudio.
```r
devtools::install_github("rstudio/leaflet#526")
```

**Comment by schloerke** [comment]
Closing as is duplicate of #525 and fixed in #526
